manim/mobject/geometry.py

```diff
@@ -807,22 +850,98 @@ def __init__(self, *args, **kwargs):
 
     @property
     def base(self):
+        r"""The base point of the arrow tip.
+        
+        This is the point connecting to the arrow line.
+
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.base.round(2)
+            array([1.65, 0.  , 0.  ])
+        
+        """
         return self.point_from_proportion(0.5)
 
     @property
     def tip_point(self):
+        r"""The tip point of the arrow tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.tip_point.round(2)
+            array([2., 0., 0.])
+        
+        """
         return self.points[0]
 
     @property
     def vector(self):
+        r"""The arrow tip vector spanning from base to tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 2, 0]), buff=0)
+            >>> arrow.tip.vector.round(2)
+            array([0.25, 0.25, 0.  ])
+        
+        """
         return self.tip_point - self.base
 
     @property
     def tip_angle(self):
+        r"""The angle of the arrow tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([1, 1, 0]), buff=0)
+            >>> round(arrow.tip.tip_angle, 5) == round(PI/4, 5)
+            True
+        
+        """
         return angle_of_vector(self.vector)
 
+    @property
+    def tip_length(self):
+        r"""The length of the arrow tip.
+
+        Examples
+        --------
+
+        ::
```

Comment:
Question: why not specify `.. code-block::` instead?

Reply:
AFAIK, it should not be necessary to do so, see https://www.sphinx-doc.org/en/1.5/markup/code.html -- but I admit, I did not try to actually generally the sphinx documentation.

Reply:
I think you're right, it's not necessary, unless you want syntax highlighting. I think we want syntax highlighting :) Please use ` code-block:: python`

Reply:
I can add it, sure -- but the link I posted says that, as python is the default highlight language, it is sufficient to put `::` and still get syntax highlighting. :-)

(I'll try it later, currently I am not home.)

Reply:
https://manim.hackl.space/reference/manim.mobject.geometry.ArrowTip.html -- Syntax highlighting works indeed, but the formatting is off. I'll investigate and try to fix it.